### ISSUE TITLE
[Frame loop] Validation error: VUID-vkAcquireNextImageKHR-semaphore-01286(ERROR / SPEC)

### ISSUE THREAD

**Issue by RenfengLiu** [google/gapid#3371]
loop the trace and got : 
```shell
17:24:45.566 I: <gapir> VUID-vkAcquireNextImageKHR-semaphore-01286(ERROR / SPEC): msgNum: 0 - vkAcquireNextImageKHR: Semaphore must not be currently signaled or in a wait state. The Vulkan spec states: If semaphore is not VK_NULL_HANDLE it must be unsignaled (https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#VUID-vkAcquireNextImageKHR-semaphore-01286)
17:24:45.566 I: <gapir>     Objects: 1
17:24:45.566 I: <gapir>         [0] 0x4b000000004b, type: 5, name: NULL

```

**Comment by AWoloszyn** [comment]
I think I may have seen this one without frame looping?

**Comment by RenfengLiu** [comment]
Fixed in PR #3400
